gatsby-config.js

```diff
@@ -431,18 +431,10 @@ module.exports = {
                         title: "Overview",
                         path: "/documentation/profile"
                     },
-                    {
-                        title: "Migration",
-                        path: "documentation/profile/migration"
-                    },
                     {
                         title: "API reference",
                         path: "documentation/profile/api-reference"
                     },
-                    {
-                        title: "Event reference",
-                        path: "documentation/profile/event-reference"
-                    },
```

Comment:
More of a question - do we not have Profile event references any more, then?

Reply:
we plan to slowly move these in their corresponding repos so we will keep them there